.github/workflows/promptflow-evals-unit-test.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           path: ${{ env.WORKING_DIRECTORY }}
       - name: install promptflow-evals from wheel
         # wildcard expansion (*) does not work in Windows, so leverage python to find and install
-        run: poetry run pip install $(python -c "import glob; print(glob.glob('promptflow_evals-*.whl')[0])")
+        run: poetry run pip install --pre $(python -c "import glob; print(glob.glob('promptflow_evals-*.whl')[0])")
```

Comment:
I think it installs from local built wheel. do we really need `--pre` here?

Reply:
This is for dependencies right now. promptflow-core is published only as a beta version package right now.

Reply:
Once we have an official release version this can be removed.